### PR TITLE
feat(TBD-12445): CVE on commons-codec below 1.10 versions

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,1053 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml:1]: dataproc was removed

File: main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml [hunk 2]
```diff
@@ -0,0 +1,350 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.hdinsight340/plugin.xml:1]: hdi 340 was removed

File: main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml [hunk 3]
```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.hdinsight360/plugin.xml:1]: hdi360 is removed

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp2xxTemplate.json [hunk 4]
```diff
@@ -0,0 +1,1600 @@
+{
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp2xxTemplate.json:1]: hdp2xx was removed

File: main/plugins/org.talend.hadoop.distribution.piv200/plugin.xml [hunk 5]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.piv200/plugin.xml:1]: piv200 was removed

File: main/plugins/org.talend.hadoop.distribution.qubole/plugin.xml [hunk 6]
```diff
@@ -0,0 +1,420 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.qubole/plugin.xml:1]: qubole was removed

File: main/plugins/org.talend.hadoop.distribution.spl30x/plugin.xml [hunk 7]
```diff
@@ -0,0 +1,870 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.spl30x/plugin.xml:1]: this plugin was removed

**Review by AlixMetivier — CHANGES_REQUESTED** [review]
some files were removed

File: main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml (hunk 1 above)
Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.dataproc14/plugin.xml]: Sorry, My bad. Resolved.

**Review by AlixMetivier — APPROVED** [review]

**Review by cbadillo1603 — APPROVED** [review]
reviewed

**Pull request merged**